### ISSUE TITLE
@@ creates {x} rather than {{x}} lot price (BZ#1031)

### ISSUE THREAD

**Issue by tbm** [ledger/ledger#1031]
Note: the issue was created automatically with bugzilla2github

Bugzilla bug ID: BZ#1031
From: Martin Michlmayr (@tbm)
CC:   @thdox

**Comment by Fuco1** [comment]
I don't think this is a bug. Ledger always shows the amount *annotations* as unit prices and that seems pretty consistent.  Specifying cost without an annotation actually creates an annotation on the amount (code is in xact.cc finalize) so it is then handled the same way in the output.
